pom.xml

```diff
@@ -26,7 +26,7 @@
   <packaging>pom</packaging>
 
   <properties>
-    <spark.version>2.3.0-SNAPSHOT</spark.version>
+    <spark.version>2.3.1-SNAPSHOT</spark.version>
```

Comment:
2.3.0-SNAPSHOT will use `spark-core_2.11-2.3.0-20180112.105540-242.jar`.
2.3.1-SNAPSHOT will follow the latest one like `spark-core_2.11-2.3.1-20180210.080833-29.jar`.